mailpoet/lib/AdminPages/PageRenderer.php

```diff
@@ -155,6 +156,7 @@ public function displayPage($template, array $data = []) {
         'automationListing' => admin_url('admin.php?page=mailpoet-automation'),
         'automationEditor' => admin_url('admin.php?page=mailpoet-automation-editor'),
         'automationTemplates' => admin_url('admin.php?page=mailpoet-automation-templates'),
+        'imageAssets' => plugin_dir_url(Env::$file) . 'assets/img/',
```

Comment:
I think we store any bigger assets under `plugin_repository/assets`, which pushes them to WP.org CDN.

That said, do we have these as SVGs? I know there's a photo in one of them, so it can be problematic. In all cases, they look blurry on a retina display so even with PNGs, we'll need a retina-friendly resolution, I guess.

As for the path, in code we seem to namespace it `automation`, not `automations`.

Reply:
I just check Figma, and they seem to be SVGs, so let's use that, if possible 🙏

Reply:
I downloaded the SVGs and because of the embedded images and base64, they grow up to 10MB. I used 2x PNGs instead: 9036139d7

Assets path change: f0bcb92c1

Reply:
after rebase
images: [f136511](https://github.com/mailpoet/mailpoet/pull/4461/commits/f136511fd82930118c214217802524ce6e2abf13)
asset path: [9e1df17](https://github.com/mailpoet/mailpoet/pull/4461/commits/9e1df17328c4c4749e7fe07aaf9c81666c3c51f4)

Reply:
[b425650](https://github.com/mailpoet/mailpoet/pull/4461/commits/b4256500773147b102c1a17854f8e0758150d0eb) uses CDN path now.